src/lib/backend.ai-client-node.js

```diff
@@ -1126,6 +1127,15 @@ class Client {
         else {
             hdrs.set('Content-Type', content_type);
         }
+        // Add secure tag if payload is encoded.
+        if (false) {
+            console.log("test");
```

Comment:
is this console.log intended??

Reply:
Ah; That js file will be disappeared... after merging.